test/testdata/dsl/rails/cattr_reader.rb

```diff
@@ -0,0 +1,25 @@
+# typed: strict
+
+class GoodUsages
+  extend T::Sig
+  cattr_reader :both, :foo
+  cattr_reader :no_instance, instance_accessor: false
+  cattr_reader :bar, :no_reader, instance_reader: false
+
+  sig {void}
+  def usages
+    both
+    no_instance # error: Method `no_instance` does not exist
+    no_reader # error: Method `no_reader` does not exist
+  end
```

Comment:
It seems like this test duplicates a bunch of stuff with `cattr_accessor`. Is that intended?

Reply:
Ohhh the actual send is different `cattr_accessor` / `cattr_reader`. nvm.

Reply:
Yea, `cattr_reader` and `cattr_writer` are the two halves of `cattr_accessor` hence the duplication.